### ISSUE TITLE
Cannot register due to limitation in registration field

### ISSUE THREAD

**Issue by MsgLosers** [dalaranwow/dalaran-wow#5670]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Website.
New User Registration.
I am unable to complete registration to join your server because of two limitations:
1. Your two e-mail fields do not accept e-mail addresses that are longer than 32 characters. My e-mail is longer than that so I cannot join.
2. You don't accept my name.

**Expected behaviour**: Tell us what should happen instead
I don't know what to tell you, these limitations are... nonsense? I hope you can see the light and have them removed so more people can join.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
screenshot:
![alt text](https://i.paste.pics/ddcd018fef0d9b41bb1b86f11078c89a.png "Proof")
https://i.paste.pics/ddcd018fef0d9b41bb1b86f11078c89a.png

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mercred** [comment]
Regarding email  this is not a bug, this is a design choice by whoever was setting up the database. I'm sure they are not going to change that simpy because someone has an email longer than 32 characters.
Also you have to write the name that was not accepted. Otherwise your issue is formulated like "I entered something into the name field and it got rejected, fix it please"

**Comment by Golganeth** [comment]
This is the in-game Bug Tracker which lists only in-game issues and not the website features.
Invalid Report, closing.

**Comment by Chenocide** [comment]
Maybe make a post about this on the website instead.
